frontend/packages/dev-console/integration-tests/features/.github/PULL_REQUEST_TEMPLATE/qe-pr.md

```diff
@@ -0,0 +1,22 @@
+**Jira Id:**
+<!-- For e.g User story Jira Id : https://issues.redhat.com/browse/ODC-XXX -->
+<!-- For Epic related gherkin scripts, e.g Epic Jira Id : https://issues.redhat.com/browse/ODC-XXX -->
+
+**Acceptance Criteria:**
+<!-- Briefly describe Acceptance criteria of Epic -->
+
+**Pre-conditions for Epic validation:**
+<!-- If any setup required while performing epic validation [which is not mentioned in Back ground section], mention the details -->
+
+**Checks required for approving PR:**
+<!-- Below criteria should met before approving the pr, use [x] -->
+- [ ] Add @epic-number to the scenarios or feature file [e.g: @odc-xxx]
+- [ ] Add @to-do for automation possible scenarios
+- [ ] Add @regression or @smoke based on the importance of the scenario
```

Comment:
Good point, I will include "@un-verified for the test scenarios not verified by ODC QE". Going forward other operators teams are also contributing test scenarios to our git repository

Reply:
It's not so much not verified by us... it's not ever tested. If you write a script and you don't execute it, that's a problem. The two scenarios I can see are:

1. It currently can work on a cluster today, and it was not tested because of the "Bystander Effect" and everyone assumed someone else tested it
2. It cannot be executed on the cluster because we are doing it in parallel with the Dev work and the feature is not merged yet

If it's 1, this is the problem we are having and I would like that gap to be closed. Any author of these Gherkin scripts, be it our team or not, should be executing them and thus should _never_ have the `@un-verified` annotation (within' reason).

If it's 2, then that's a good reason for us to use this label. That way we can swing back when doing epic verification or once at the end of the sprint to test all of them and remove the `@un-verified` annotation.

I hope this is clear as it's the foundation for all this effort. If you have any questions or concerns, please let me know.

Reply:
That makes sense